app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/tasklist/multiple_disposals_task_list.scala.html

```diff
@@ -200,6 +203,15 @@ <h2 class="app-task-list__section">
 
     </ol>
 
+    @{{if(fillingOutReturn.isAmendReturn)
+        cancelOrContinue(
+            messages("button.cancel"),
+            returns.amend.routes.AmendReturnController.confirmCancel(AmendReturnController.ConfirmCancelBackLocations.checkAnswers).url,
```

Comment:
`ConfirmCancelBackLocations.checAnswers` isn't the one you want here - that refers to the amend cya page. You should create a new back link location called something like `taskList` and point it to the task list endpoint